textproc/fribidi/Portfile

```diff
@@ -19,34 +20,14 @@ long_description \
 
 homepage        http://fribidi.org/
 
-checksums       rmd160  c8d45104684a3093deced1961654e6c31ef02149 \
-                sha256  9e30a05d4691622bd5dd3f34cdff8fa323b3b425e50d1968d5ed6da35f894e9f \
-                size    2087866
+checksums       rmd160  2bd97363d956db9bfe7f552ba13b390c3330a1e2 \
+                sha256  c5e47ea9026fb60da1944da9888b4e0a18854a0e2410bbfe7ad90a054d36e0c7 \
+                size    1141684
+use_xz          yes
 
 depends_build   port:pkgconfig
 
-use_autoreconf  yes
-autoreconf.args -fvi
```

Comment:
The build fails because of:

`'automake-1.16' is missing on your system.
`

Probably because these 2 lines were removed. You probably either need to restore them or at least add a build dependency on `automake`.